velox/functions/prestosql/StringFunctions.cpp

```diff
@@ -153,36 +153,49 @@ class ConcatFunction : public exec::VectorFunction {
       const TypePtr& /* outputType */,
       exec::EvalCtx& context,
       VectorPtr& result) const override {
-    VectorPtr emptyVectorPtr;
-    prepareFlatResultsVector(result, rows, context, emptyVectorPtr);
```

Comment:
Don't we use prepareFlatResultsVector to possibly reuse input vectors?

Reply:
There is not much to reuse here since the main cost is in allocating and writing new strings. Hence, I removed it to make the code a tiny bit simpler.